rollup.config.js

```diff
@@ -78,6 +79,34 @@ function strings({ include, exclude } = {}) {
   };
 }
 
+// Inject deno.arch/deno.platform from Node's process.arch/process.platform
+function platform({ include, exclude } = {}) {
+  if (!include) {
+    throw new Error("include option must be passed");
+  }
+
+  const filter = createFilter(include, exclude);
+
+  return {
+    name: "platform",
+    /**
+     * @param {any} _code
+     * @param {string} id
+     */
+    transform(_code, id) {
+      if (filter(id)) {
+        // arch and platform comes from Node
+        return {
+          code: `export const arch = "${process.arch}";
+          export const platform = "${process.platform}";
+          `,
+          map: { mappings: "" }
```

Comment:
I think it is fine for now, but we might want a todo to actually have a valid map in here, as it might cause confusing stack traces.  It may be worth testing it to just see how bad it is at the moment.

Reply:
So I just ripped off the sourceMap generating code from `rollup-plugin-inject`. Hopefully it could work